package.json

```diff
@@ -12,7 +12,7 @@
 	"license": "MIT",
 	"icon": "public/icon.png",
 	"engines": {
-		"vscode": "^1.19.0"
+		"vscode": "^1.54.0"
```

Comment:
Is there a reason why vscode engine was mismatched from `@types/vscode": "^1.54.0` below? I updated the engine instead of updating the types since I was able to run the package command successfully.

Reply:
found the same issue. already updated it on master. just pull or rebase.